napari/layers/points/_tests/test_points.py

```diff
@@ -1678,8 +1678,8 @@ def test_interaction_box():
 def test_world_data_extent():
     """Test extent after applying transforms."""
     data = [(7, -5, 0), (-2, 0, 15), (4, 30, 12)]
-    min_val = (-2, -5, 0)
-    max_val = (7, 30, 15)
+    min_val = (-7, -10, -5)
+    max_val = (12, 35, 20)
     layer = Points(data)
```

Comment:
I think it would be more clear if we explicitly set the point size.
```suggestion
    layer = Points(data, size=10)
```